zuko-ui/src/components/PostShareLinks/PostShareLinks.jsx

```diff
@@ -1,25 +1,19 @@
 import styles from './PostShareLinks.module.scss';
 
 export default function PostShareLinks() {
+	const href = window.location.href;
+
 	const data = [
```

Comment:
Maybe better to store these static data and the whole logic of generating the links for sharing outside the component